shared/.pronto_spell.pws

```diff
@@ -274,6 +275,7 @@ hanna
 hapikey
 hashchange
 hashie
+hashmap
```

Comment:
I thought this name is not finalized, I’d want us to revise that implementation with a global state and come up with a better name and a better Liquid API.

Others LG.

Reply:
it doesn't, but I would like to move the task further into QA and not wait a few weeks until a proper solution is implemented. Having 30+ messages on every CI run is way worse, than a single word in pronto checks.